frontend/package.json

```diff
@@ -33,10 +32,10 @@
         "react-google-charts": "^4.0.0",
         "react-grid-layout": "^1.2.5",
         "react-idle-timer": "^4.6.4",
-        "react-scripts": "^4.0.3",
-        "styled-components": "^5.3.0",
+        "react-scripts": "^5.0.1",
         "typescript": "^4.3.4",
-        "use-deep-compare-effect": "^1.6.1"
+        "use-deep-compare-effect": "^1.6.1",
+        "yarn": "^1.22.19"
```

Comment:
It's strange to have `yarn` here. We should probably just switch over to yarn fully, and remove package-lock.json, if that's what you'd like to use to manage the packages

Reply:
can you help me on making sure to clean up these? thank you very much

Reply:
See 3186ff7